bears/c_languages/AStyleLintBear.py

```diff
@@ -0,0 +1,62 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='astyle',
+        output_format='corrected',
+        use_stdin=True)
+class AStyleLintBear:
+    """
+    Artistic Style is a source code indenter, formatter,
+    and beautifier for the C, C++, C++/CLI, Objective‑C,
+    C# and Java programming languages.
+
+    For more information, consult <http://astyle.sourceforge.net/astyle.html>.
+    """
+
+    LANGUAGES = {'C', 'C++', 'Objective-C', 'C#', 'Java'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='astyle')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         astyle_style: str='',
+                         use_spaces: bool=None,
+                         astyle_config: str='',
+                         indent_size: int=0):
+        """
+        :param astyle_style:
```

Comment:
I think its just the bracket style. At least thats what the documentation says. But astyle itself just call it `style` and the command line parameter is `style`